pkg/cmd/system/events.go

```diff
@@ -48,18 +48,101 @@ type EventOut struct {
 type Status string
 
 const (
-	START   Status = "START"
-	UNKNOWN Status = "UNKNOWN"
+	START   Status = "start"
+	UNKNOWN Status = "unknown"
 )
 
+var statuses = [...]Status{START, UNKNOWN}
+
+func isStatus(status string) bool {
+	status = strings.ToLower(status)
+
+	for _, supportedStatus := range statuses {
+		if string(supportedStatus) == status {
+			return true
+		}
+	}
+
+	return false
+}
+
 func TopicToStatus(topic string) Status {
-	if strings.Contains(strings.ToUpper(topic), string(START)) {
+	if strings.Contains(strings.ToLower(topic), string(START)) {
 		return START
 	}
 
 	return UNKNOWN
 }
 
+// EventFilter for filtering events
+type EventFilter func(*EventOut) bool
+
+// Similar to podman implementation:
```

Comment:
In Golang, usually, the comments should start with the Function Name. When documenting a function, the first sentence should typically start with the function's name. This format helps quickly clarify the function’s purpose.

Reply:
I changed the comments. Let me know if that works.